requirements-tests.txt

```diff
@@ -1,6 +1,6 @@
 av
 coveralls
-neo>=0.10.0
+neo>=0.6.0
```

Comment:
Maybe for the test we could explicitly keep >=0.10.0 because it is the one that will be tested.

Reply:
Agreed. (And besides, SpikeInterface requires Neo>=0.10.0 anyway.)